attributes/default.rb

```diff
@@ -35,9 +35,20 @@
 default['hops']['conf_dir']                    = node['hops']['conf_dir_parent'] + "/hadoop"
 default['hops']['share_dir']                    = node['hops']['base_dir'] + "/share/hadoop"
 
-default['hops']['dn']['data_dir']              = "file://" + node['hops']['data_dir'] + "/hdfs/dn"
-default['hops']['dn']['data_dir_permissions']  = '700'
-default['hops']['nn']['name_dir']              = "file://" + node['hops']['data_dir'] + "/hdfs/nn"
+default['hops']['enable_cloud_storage']        = false
+default['hops']['cloud_provider']              = "AWS"
+default['hops']['aws_s3_region']              = "eu-west-1"
+default['hops']['aws_s3_num_buckets']         = "10"
+default['hops']['aws_s3_bucket_prefix']       = "hopsfs.bucket"
+default['hops']['cloud_bypass_disk_cache']         = false 
+default['hops']['cloud_max_upload_threads']        = "20"
+default['hops']['cloud_store_small_files_in_db']   = true 
+
+default['hops']['dn']['data_dir']                       = "file://" + node['hops']['data_dir'] + "/hdfs/dn/disk"
```

Comment:
We should be careful about his attribute for upgrades as it will re-template `hdfs-site.xml` if we don't set it in the cluster definition @SirOibaf